ios/ApiPreferences.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  ApiPreferences.swift
+//  magnet
+//
+//  Created by sam on 11/7/16.
+//  Copyright © 2016 Mozilla. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+class ApiPreferences: ApiBase {
+  
+  private static let PATH = "content://preferences"
+  private var preferences: Preferences!
```

Comment:
Agree, this patch could be a lot simpler if we re-use our simple key/value `RequestStore`.